src/deploy/alternative/kubernetes-dashboard.yaml

```diff
@@ -41,11 +41,16 @@ rules:
 - apiGroups: [""]
   resources: ["secrets"]
   verbs: ["create", "watch"]
+  # Allow Dashboard to get, update and delete 'kubernetes-dashboard-key-holder' secret.
```

Comment:
We can change this description to something like `Allow Dashboard to get, update and delete Dashboard exclusive secrets`.